src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star-active.svg) no-repeat;
```

Comment:
it will be more logical if you set non-active star by default (because the count of active stars less then non-active)